bot/cogs/bigbrother.py

```diff
@@ -73,10 +77,20 @@ def update_cache(self, api_response: List[dict]):
             data = await response.json()
             self.update_cache(data)
 
-    async def get_watch_information(self, user_id: int) -> WatchInformation:
+    async def update_watched_users(self):
+        async with self.bot.http_session.get(URLs.site_bigbrother_api, headers=self.HEADERS) as response:
+            if response.status == 200:
+                data = await response.json()
+                self.update_cache(data)
+                log.trace("Updated watch list cache")
```

Comment:
this log could be improved. without the context of big brother, it might be a little hard to understand exactly what this means.
```suggestion
                log.trace("Updated Big Brother watchlist cache")
```